### PR TITLE
[Enhancement] Export/Import Routines for Fields

### PR DESCRIPTION
**Pull request by stetie**
io.py contains two function, which will export or import Field objects to a file in numpy's binary format.

### CODE REVIEW

File: postpic/io.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with postpic. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stefan Tietze, 2017
+
+import numpy as np
+from .datahandling import Axis, Field
+
+def _export_field_npy(field, filename, compressed=True):
```
Comment (skuschel) [inline postpic/io.py]: instead of compression=True, maybe better: `savefunc=np.savez_compressed`.

File: postpic/io.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    '''
+    Export a Field object including all metadata and axes to a file.
+    The file will be in the numpy binary format (npz).
+    '''
+
+    savefunc = np.savez_compressed
+
+    if not compressed:
```
Comment (skuschel) [inline postpic/io.py]: `savefunc = np.savez_compressed if compressed else np.savez`
und die code zeile direkt savefunc(...)

**Comment by stetie** [comment]
this solves points 1-3 of issue #114

File: test/test_io.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    def setUp(self):
+        pp.chooseCode('DUMMY')
+        self.dump = pp.datareader.readDump(100)
+        self.testfield = self.dump.Ey()
+        io._export_field_npy(self.testfield, 'test.npz')
+        self.testfield2 = io._import_field_npy('test.npz')
+
+    def test_data(self):
+        assert np.all(np.isclose(self.testfield.matrix, self.testfield2.matrix))
```
Comment (skuschel) [inline test/test_io.py]: self.assertTrue

**Pull request merged**